### PR TITLE
Check if Python 3.10.0 available for Mac and Windows.

### PR DESCRIPTION
**Pull request by charris**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -46,6 +46,15 @@ jobs:
           PYTHON_ARCH: 'x64'
           NPY_USE_BLAS_ILP64: 1
           BITS: 64
+        py_3.10_32:
+          PYTHON_VERSION: "3.10-dev"
```
Comment (tupui) [inline azure-pipelines.yml:50]: If it's not available yet on Azure, we should still be able to build wheels with the following no?
```suggestion
          AZURE_PYTHON_VERSION: "3.9"
          PYTHON_VERSION: "3.10"
```
```suggestion
          PYTHON_VERSION: "3.10-dev"
```

**Comment by rgommers** [comment]
Conclusion is "not available"?

**Comment by hugovk** [comment]
Correct, the `-dev` versions are installed on-demand, and on-demand isn't available for Azure Pipelines, unlike GitHub Actions (also the reason `3.10` isn't available for AZP until pre-cached images are deployed).

* *https://github.com/actions/virtual-environments/issues/4226#issuecomment-934255087

Here's the feature request to 👍 upvote for AZP to be able to install on-demand:

* https://github.com/microsoft/azure-pipelines-tasks/issues/13319

(Upvoting suggested by a GitHub employee, so they must pay attention to upvotes: 
https://github.com/actions/virtual-environments/issues/1740#issuecomment-705363333)

**Comment by charris** [comment]
> Conclusion is "not available"?

I deleted this in favor of #135. As of yesterday evening, Windows 64 bits works, Windows 32 bits was missing, and Mac needed `Python.h`. Not sure about the last, it might have been moved...

**Pull request closed without merging**